components/~common/Skip.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+function Skip() {
+  return (
+    <S>
+      <p>Skip</p>
+      <div className="dash" />
```

Comment:
maybe for the future `Skip` show be injected as a prop in the component for more reusability

Reply:
Yeah, I was thinking the same, or maybe we just wrap it in a link component? 🤔

Reply:
yes.. it will work both ways I think 😃 , we'll see about this after the app will get bigger 👍